### ISSUE TITLE
version > v15.0.0: cold start of large Angular / Tailwind CSS site shows considerable delay between "listening" and actually "serving"

### ISSUE THREAD

**Issue by bjornharvold** [nrwl/nx#12768]
## Current Behavior
In v15.0.0, both my Angular / Bootstrap apps and my Angular / Tailwind CSS apps serve up find in development. When upgrading to v15.0.1, Angular / Bootstrap apps run fine when running nx serve. However, the Angular / Tailwind CSS apps now hang and Chrome errors out with the dinosaur. There is even some randomness around this where the Angular / Tailwind CSS apps do run... but mostly not. When I downgrade to 15.0.0, all apps serve fine again.

## Expected Behavior
npx nx serve should serve up the app.

## Steps to Reproduce
- Create an Nx Angular workspace
- Add TailwindCSS
- Run npx nx serve

We run a 1300+ module monorepo. Reproducing the error in a simple monorepo example w a couple of components might not work.

### Failure Logs
None

### Environment
```
Node : 16.18.0
   OS   : darwin arm64
   yarn : 1.22.19
   
   nx : 15.0.1
   @nrwl/angular : 15.0.1
   @nrwl/cypress : 15.0.1
   @nrwl/detox : Not Found
   @nrwl/devkit : 15.0.1
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 15.0.1
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 15.0.1
   @nrwl/js : 15.0.1
   @nrwl/linter : 15.0.1
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/rollup : 15.0.1
   @nrwl/schematics : Not Found
   @nrwl/storybook : 15.0.1
   @nrwl/web : 15.0.1
   @nrwl/webpack : 15.0.1
   @nrwl/workspace : 15.0.1
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
   	 @fortawesome/angular-fontawesome: 0.11.1
   	 @ng-bootstrap/ng-bootstrap: 13.1.0
   	 @ngneat/transloco: 4.1.1
   	 @ngrx/component: 14.3.2
   	 @ngrx/component-store: 14.3.2
   	 @ngrx/data: 14.3.2
   	 @ngrx/effects: 14.3.2
   	 @ngrx/entity: 14.3.2
   	 @ngrx/router-store: 14.3.2
   	 @ngrx/store: 14.3.2
   	 @ngrx/store-devtools: 14.3.2
   	 angular-calendar: 0.30.1
   	 @compodoc/compodoc: 1.1.19
   	 @jscutlery/semver: 2.27.1
   	 @ngrx/schematics: 14.3.2
   	 @storybook/angular: 6.5.12
```

**Comment by bjornharvold** [comment]
FYI This is still an issue w v15.0.3. 15.0.0 is the highest version that works.

**Comment by Coly010** [comment]
I can't reproduce this on a clean workspace at 15.0.1, or when I create a workspace at 15.0.0 and upgrade it to 15.0.1.

Do investigate this more, I'm going to need a reproduction repo. I know you've mentioned your current workspace is very large, but until I can get a valid reproduction, I'm not going to be able to investigate what the failure you're encountering is.

**Comment by bjornharvold** [comment]
Following up on this. I upgraded to v15.0.4 to try and see if this got resolved. The issue is still there but the issue is actually different than the ticket makes it out to be. 

Once I serve up the app and the app comes back to me saying:

`** Angular Live Development Server is listening on ..... **`

There is about a 30s lag between the time the app says it's listening and when it's actually serving. Guessing the 30s is due to the size of our codebase. Now I don't know what is happening during those 30s. Chrome comes back to me with the dinosaur and says unresponsive app. But when the app is actually ready, it serves the app.

One other thing: This occurs only on a cold start and not after a change has been made to the codebase while it's already serving the app.

Hope this sheds some 💡 on things. Will rename the issue as well.
